package.json

```
@@ -43,7 +43,7 @@
     "jsdoc-to-markdown": "^7.1.1"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=20"
```

18 is still being maintained so we can include that here.

```suggestion
    "node": ">=18"
```